source/wireversion-featurelist.rst

```diff
@@ -0,0 +1,52 @@
+====================================
+Server Wire version and Feature List
+====================================
+
+.. list-table::
+   :header-rows: 1
+
+   * - Server version
+     - Wire version
+     - Feature List
+
+   * - 2.6
+     - 1
+     - | Aggregation cursor
+       | Auth commands
+
+   * - 2.6 
+     - 2
+     - | Write commands (insert/update/delete)
+       | Aggregation $out pipeline operator
+  
+   * - 3.0
+     - 3
+     - | List collections
+       | List indexes
```

Comment:
listCollections command, listIndexes command.

Reply:
Updated and added listDatabases to 3.6 list